server/meta.go

```diff
@@ -31,7 +25,7 @@ func (h metaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.mux.ServeHTTP(w, r)
 }
 
-func newMetaHandler(opt Option) metaHandler {
+func NewMetaHandler(opt Option) metaHandler {
```

Comment:
__[golint]__
* [golint] exported function NewMetaHandler should have comment or be unexported
* [golint] exported func NewMetaHandler returns unexported type server.metaHandler, which can be annoying to use

<!-- Comment by [SideCI](https://sideci.com) -->